lib/hbs-packer.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path-posix');
+const CachingWriter = require('broccoli-caching-writer');
+const {
+  readFile, svgDataFor, toPosixPath, saveToFile, relativePathFor
+} = require('./utils');
+
+function ensureDirectoryExistence(filePath) {
+  const dirname = path.dirname(filePath);
+  if (fs.existsSync(dirname)) {
+    return;
+  }
+  ensureDirectoryExistence(dirname);
+  fs.mkdirSync(dirname);
+}
+
+
+const templateOnlyComponent = (svg) => `
+  import templateOnly from '@ember/component/template-only';
+  import { setComponentTemplate } from '@ember/component';
+  import { hbs } from 'ember-cli-htmlbars';
+
+  export default setComponentTemplate(hbs\`${svg}\`, templateOnly());
```

Comment:
@betocantu93 does it mean templates won't be pre-compiled at build time and instead compiled at run time once they are loading on client-side?

Reply:
Nop, they are compiled at build time but each svg in a separate file, I'm using treeForAddon to accomplish this